unlock-app/src/config.js

```diff
@@ -4,6 +4,9 @@ import {
   CANONICAL_BASE_DEV_URL,
   CANONICAL_BASE_STAGING_URL,
   CANONICAL_BASE_URL,
+  CANONICAL_STATIC_DEV_URL,
+  CANONICAL_STATIC_STAGING_URL,
+  CANONICAL_STATIC_URL,
```

Comment:
See comment above but I think this is not the right approach. We need a single `CANONICAL_STATIC_URL` set as an env variable